docs/source/public/overview.rst

```diff
@@ -180,11 +181,15 @@ Here is an example of a very simple script that will create and submit a separat
      relpath=${file:14}  # strip off "network_setup/"
```

Comment:
I think the line above needs to be change to this: 

```bash
 for file in network_setup/transformations/*.json; do
```

not sure if the rest of the rel-paths work then?

Reply:
I don't have a strong view here - possibly putting thing sin a "transformations" folder might be useful so the top level directory is cleaner. That being said, I consider this to be reasonably low priority so I'm leaving it up to you @hannahbaumann

Reply:
Ok! I can change it, just to be more similar to the regular CLI output, so that people could reuse the same run script if they used the regular CLI to create their networks.

Reply:
Edit: I just realized that I would also have to change the run script to create the extra directory and I'm not sure if it's worth it since besides the .jsons files there's only the ligand_network.graphml file in the folder. So maybe we do leave it as is, unless others think it would be beneficial to be consistent @IAlibay @RiesBen

Reply:
If it means not adding any last minute work, I am more than happy to go with the current state of things @hannahbaumann !

Reply:
Hi, I think it would be great if our folder structure follows this here: 
https://docs.openfree.energy/en/stable/tutorials/rbfe_cli_tutorial.html

in my understanding the final folder structure, after full execution should look like this:

some_benchmark_folder

--network_setup
-----ligand_*json
-----transformations
-------*jsons

--results
----fun1
------shared_*
--------*nc
-----fun1.json
----fun2
------shared_*
--------*nc
-----fun2.json
---- ...

--final_results.tsv

Reply:
will transfer to an issue